constants.py

```diff
@@ -36,9 +36,8 @@
 NET_POL_ORDER = 1000
 
 # The priority assigned to the backstop policy that applies
-# to traffic which doesn't match one of the configured policies
-# in the NET_POL_TIER_NAME tier.
-NET_POL_BACKSTOP_ORDER = 2000
+# to traffic which doesn't match one of the configured policies.
+NET_POL_NO_MATCH_ORDER = int(os.getenv("NO_MATCH_ORDER", 2000))
```

Comment:
Should this be prefixed to indicate k8s policy, and to reduce risk of possible clash with other apps using the same name?  Does k8s policy honor any other environment variables?  If so, is this consistent with how those other ones are named?